xtask/codegen/src/generate_nodes.rs

```diff
@@ -1174,6 +1174,6 @@ pub(crate) fn group_fields_for_ordering(node: &AstNodeSrc) -> Vec<Vec<&Field>> {
 pub fn should_token_be_quoted(token: &str) -> bool {
     matches!(
         token,
-        "$=" | "$_" | "U+" | "<![CDATA[" | "]]>" | "   " | "_" | "__" | "`" | "```"
+        "$=" | "$_" | "U+" | "<![CDATA[" | "]]>" | "   " | "_" | "__" | "`" | "```" | "{{" | "}}"
```

Comment:
these should probably be in there too?
```suggestion
        "$=" | "$_" | "U+" | "<![CDATA[" | "]]>" | "   " | "_" | "__" | "`" | "```" | "{{" | "}}" | "{" | "}"
```

Reply:
No, there's another place that takes care of it, so we don't need to add it